app/build.gradle.kts

```diff
@@ -61,6 +61,8 @@ android {
     defaultConfig {
         applicationId = "com.neo.speaktouch"
 
+        configure<BasePluginExtension> { archivesName.set(rootProject.name) }
```

Comment:
Where did this code go?

Reply:
I'm not sure what you mean but it sets base apk name. Previously, would like to just add `setProp("archiveBaseName", rootProject.name)` but it's deprecated according to [this](https://docs.gradle.org/8.5/userguide/upgrading_version_8.html#base_convention_deprecation)